app/models/collection_template.rb

```diff
@@ -76,6 +76,11 @@ def fingerprint_pathselection
     "#{image_paths_to_hex.join('_')}"
   end
 
+  def postprocessed_image
+    "#{image.file_name_no_extension}_"\
+    'postprocess_tmp'
```

Comment:
We may want to fingerprint this at some point, using an MD5 of the skeletonize options. The fingerprint should help with image cache invalidation. Not suggesting this here, but leaving the comment for future sake. Likely ridge detection will inform some of this.